creating_images/sti.adoc

```diff
@@ -49,12 +50,12 @@ See the following diagram for the basic S2I build workflow:
 .Build Workflow
 image::sti-flow.png[S2I workflow]
 
-////
-* Run build's responsibility is to untar the sources, scripts and artifacts (if such exist) and invoke `assemble` script. If this is second run (after catching `tar`/`/bin/sh` error) it's responsible only for invoking `assemble` script, since both scripts and sources are already there.
-////
+* Run build's responsibility is to untar the sources, scripts and artifacts (if such exist) and invoke `assemble` script. If this is second run (after catching `tar`/`/bin/sh` error) it is responsible only for invoking `assemble` script, since both scripts and sources are already there.
```

Comment:
- "and invoke the `assemble` script."
- "If this is the second run"
- Does "after catching `tar`/`/bin/sh` error" mean "after catching any `tar` or `/bin/sh` errors"?